src/processor/map/map_copy_from.cpp

```diff
@@ -107,13 +96,14 @@ std::unique_ptr<PhysicalOperator> PlanMapper::mapCopyNodeFrom(
     }
     const auto numWarningDataColumns = copyFromInfo->source->getNumWarningDataColumns();
     KU_ASSERT(columnTypes.size() >= numWarningDataColumns);
-    auto info = std::make_unique<NodeBatchInsertInfo>(nodeTableEntry,
-        storageManager->compressionEnabled(), std::move(columnIDs), std::move(columnTypes),
+    auto info = std::make_unique<NodeBatchInsertInfo>(nullptr, // placeholder
```

Comment:
Ditto here on changing the constructor instead.

Reply:
Fixed with acc6017393f9fe85c42fbe7ccb02d426b2b954b4